pkg/controllers/dynakube/logmodule/reconciler.go

```diff
@@ -38,11 +45,23 @@ func NewReconciler(clt client.Client,
 		configSecretReconciler:           configsecret.NewReconciler(clt, apiReader, dk),
 		daemonsetReconciler:              daemonset.NewReconciler(clt, apiReader, dk),
 		oneAgentConnectionInfoReconciler: oaconnectioninfo.NewReconciler(clt, apiReader, dtc, dk),
+		monitoredEntitiesReconciler:      monitoredentities.NewReconciler(dtc, dk),
 	}
 }
 
 func (r *Reconciler) Reconcile(ctx context.Context) error {
-	err := r.oneAgentConnectionInfoReconciler.Reconcile(ctx)
+	err := r.monitoredEntitiesReconciler.Reconcile(ctx)
+	if err != nil {
+		return err
+	}
+
+	if r.dk.Status.KubernetesClusterMEID == "" || r.dk.Status.KubernetesClusterName == "" {
```

Comment:
What do you think about abstracting this? either both fields abstract on their own or abstract the whole check? Or do you think this is too much?